pages/export.vue

```diff
@@ -57,6 +57,17 @@
         });
     }
 
+    useHead({
+        link: [
+            {
+                rel: 'preload',
+                as: 'image',
+                href: './RatsInLove.png',
+                type: 'image/png',
+            },
+        ],
+    });
+
```

Comment:
Locally the Rats load fine for me without this. I also think this has to work without this, since there is no reason to why I would work different for this particular image.

Reply:
Did you try loading the image just with `/` instead of `./`.

Reply:
Yes! You're right about not needing the ./! However it does not load on my phone without the useHead.. 
![IMG_5980](https://github.com/comixcraft/Editor/assets/107397144/0683375b-d733-42a2-990d-17d50c885709)
![IMG_5981](https://github.com/comixcraft/Editor/assets/107397144/888c5119-ad97-49e4-a754-a5b60070d521)

Reply:
One was in Safari and the other one in chrome.

Reply:
I'm also a bit confused why this image is not loading for this in my phone without this useHead, is it a problem to use it?

Reply:
Normally you should not need it to load the images. Us using it disguises the underlying problem of what is working wrong.

Reply:
But I see that we now spent already a lot of time on this task, this most probably also seems to be a deployment issue from netlify.

I would maybe just ask to move the image loading from the useHead in the page to the nuxt-config. https://nuxt.com/docs/api/nuxt-config#head